pom.xml

```diff
@@ -67,8 +67,8 @@ limitations under the License.
         <flink.reuseForks>true</flink.reuseForks>
 
         <!-- dependencies versions -->
-        <flink.version>1.19.1</flink.version>
-        <flink.major.version>1.19</flink.major.version>
+        <flink.version>1.20.0</flink.version>
```

Comment:
@ferenc-csaky Could you bump this version to 1.20.1 as we just released it?

Reply:
That was my original idea as well, but that generated more and more changes (docs, CI, etc), and 1.19.1 has to be bumped to 1.19.2 as well anyways. I'm happy to update both patch versions in a follow-up PR in 1 chunk. WDYT?

Reply:
As they're hotfix versions, follow-up PR makes sense to me, thanks for your quick response.